ixdiagnose/artifacts/logs.py

```diff
@@ -0,0 +1,20 @@
+from .base import Artifact
+from .items import Directory, File
+
+
+class Logs(Artifact):
+    base_dir = '/var/log'
+    name = 'logs'
+    individual_item_max_size_limit = 10 * 1024 * 1024
+    items = [
+        Directory('openvpn'),
+        Directory('pods'),
+        Directory('samba4'),
+        File('daemon.log'),
+        File('failover.log'),
+        File('kern.log'),
+        File('messages'),
+        File('middlewared.log'),
```

Comment:
We should go ahead and add regex filtering to the `File` class because, for example, `middlewared.log` is the active file but we roll those over every 10MB and keep up to 10 (I think). They're named `middlewared.log.0`, `middlewared.log.1`, `middlewared.log.2` etc. Same for `zettarepl` and probably more but those are the ones I remember off top of my head.